test/tests/optimizationreporter/objective_gradient_minimize/bc_load_linearFunction/plot_iterations.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+import pandas as pd
+import os
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+import math
+
+SMALL_SIZE = 8
+MEDIUM_SIZE = 10
+BIGGER_SIZE = 12
+
+plt.rc('font', size=SMALL_SIZE)          # controls default text sizes
+plt.rc('axes', titlesize=SMALL_SIZE)     # fontsize of the axes title
+plt.rc('axes', labelsize=SMALL_SIZE)    # fontsize of the x and y labels
+plt.rc('xtick', labelsize=SMALL_SIZE)    # fontsize of the tick labels
+plt.rc('ytick', labelsize=SMALL_SIZE)    # fontsize of the tick labels
+plt.rc('legend', fontsize=7)    # legend fontsize
+plt.rc('figure', titlesize=SMALL_SIZE)  # fontsize of the figure title
+#-------------------------------------------------------------------------------
+df_nls = pd.read_csv('nls_optInfo_0001.csv')
+df_cg = pd.read_csv('cg_optInfo_0001.csv')
+df_nm = pd.read_csv('nm_optInfo_0001.csv')
+df_lmvm = pd.read_csv('lmvm_optInfo_0001.csv')
+print('fracture dataframe description: ',df_nls.columns.values)
```

Comment:
If we are going to add a python utility it needs to be general, like using input arguments instead of hardcoded filenames. Also, using json format with ReporterReader would be way cleaner.

Reply:
@mguddati you can delete this python file.  I added it but it doesn't belong in the test directory.  I'm fine with json output, I haven't tried it but it should work

Reply:
done